src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -75,11 +75,12 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
             return;
         }
 
-        this.updateMapWithConfig(prevConfig);
-
         if (selectedSegmentItems && !isEqual(selectedSegmentItems, prevSelectedSegmentItems)) {
             this.setFilterMap();
+            return;
```

Comment:
we can combine this :)
`return this.setFilterMap(); `

Reply:
fixed